website/templates/emails/pending_retraction.txt.mako

```diff
@@ -0,0 +1,14 @@
+Hello ${user.fullname},
```

Comment:
basically the same comment again, first line:

"This email is to notify you that another administrator has initiated a retraction for.."

but if we want to go with a more conversational tone, I don't object.